2018-09-29 17:38:35.445 5000 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-29 17:38:35.448 5000 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.003s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-29 17:39:35.445 5000 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-29 17:39:35.449 5000 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.003s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-29 17:40:22.080 5000 DEBUG oslo_concurrency.lockutils [req-06882cfd-0257-4eff-b3ed-d3084c938506 8fbe107dce56405f89cba07bca41fa33 d0066f73b1db4dcabfc61318ae559960 - - -] Acquired semaphore "dhcp-agent-network-lock-a3127656-d5cf-4734-b868-3ae134f10b1f" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 17:40:22.790 5000 DEBUG neutron.agent.dhcp.agent [req-bccc391a-251d-4706-adcd-6b3bf1f358dd - - - - -] Calling driver for network: a3127656-d5cf-4734-b868-3ae134f10b1f action: enable call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-29 17:40:22.795 5000 DEBUG neutron.agent.linux.utils [req-bccc391a-251d-4706-adcd-6b3bf1f358dd - - - - -] Unable to access /var/lib/neutron/dhcp/a3127656-d5cf-4734-b868-3ae134f10b1f/pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 17:40:22.797 5000 DEBUG neutron.agent.linux.dhcp [req-bccc391a-251d-4706-adcd-6b3bf1f358dd - - - - -] DHCP port dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-a3127656-d5cf-4734-b868-3ae134f10b1f on network a3127656-d5cf-4734-b868-3ae134f10b1f does not yet exist. Checking for a reserved port. _setup_reserved_dhcp_port /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1272
2018-09-29 17:40:22.798 5000 DEBUG neutron.agent.linux.dhcp [req-bccc391a-251d-4706-adcd-6b3bf1f358dd - - - - -] DHCP port dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-a3127656-d5cf-4734-b868-3ae134f10b1f on network a3127656-d5cf-4734-b868-3ae134f10b1f does not yet exist. Creating new one. _setup_new_dhcp_port /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1293
2018-09-29 17:40:25.742 5000 DEBUG neutron.agent.linux.utils [req-0232ade9-fa7c-4a5d-a08d-f20007037a24 - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 17:40:25.837 5000 DEBUG neutron.agent.linux.utils [req-0232ade9-fa7c-4a5d-a08d-f20007037a24 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'add', 'qdhcp-a3127656-d5cf-4734-b868-3ae134f10b1f'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 17:40:25.874 5000 DEBUG neutron.agent.linux.utils [req-0232ade9-fa7c-4a5d-a08d-f20007037a24 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-a3127656-d5cf-4734-b868-3ae134f10b1f', 'sysctl', '-w', 'net.ipv4.conf.all.promote_secondaries=1'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 17:40:25.905 5000 DEBUG neutron.agent.linux.utils [req-0232ade9-fa7c-4a5d-a08d-f20007037a24 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-a3127656-d5cf-4734-b868-3ae134f10b1f', 'ip', 'link', 'set', 'lo', 'up'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 17:40:25.928 5000 DEBUG neutron.agent.linux.utils [req-0232ade9-fa7c-4a5d-a08d-f20007037a24 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-a3127656-d5cf-4734-b868-3ae134f10b1f', 'sysctl', '-w', 'net.ipv6.conf.default.accept_ra=0'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 17:40:25.955 5000 DEBUG neutron.agent.linux.utils [req-0232ade9-fa7c-4a5d-a08d-f20007037a24 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-a3127656-d5cf-4734-b868-3ae134f10b1f', 'ip', 'link', 'set', 'tap25b2c0ed-0a', 'up'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 17:40:25.992 5000 DEBUG neutron.agent.linux.utils [req-0232ade9-fa7c-4a5d-a08d-f20007037a24 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-a3127656-d5cf-4734-b868-3ae134f10b1f', 'ip', '-o', 'link', 'show', 'tap25b2c0ed-0a'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 17:40:26.020 5000 DEBUG neutron.agent.linux.utils [req-0232ade9-fa7c-4a5d-a08d-f20007037a24 - - - - -] Running command: ['ip', '-o', 'link', 'show', 'br-int'] create_process /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:92
2018-09-29 17:40:26.582 5000 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=0): DelPortCommand(if_exists=True, bridge=None, port=tap25b2c0ed-0a) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-09-29 17:40:26.584 5000 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Transaction caused no change do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:114
2018-09-29 17:40:26.587 5000 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=0): AddPortCommand(bridge=br-int, may_exist=False, port=tap25b2c0ed-0a) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-09-29 17:40:26.589 5000 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=1): DbSetCommand(table=Port, col_values=(('tag', 4095),), record=tap25b2c0ed-0a) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-09-29 17:40:26.592 5000 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=2): DbSetCommand(table=Interface, col_values=(('type', 'internal'), ('external_ids', {'iface-id': u'25b2c0ed-0a35-4108-b73e-50a5ab73a807', 'iface-status': 'active', 'attached-mac': u'fa:16:3e:25:03:7b'})), record=tap25b2c0ed-0a) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-09-29 17:40:26.627 5000 DEBUG neutron.agent.linux.utils [req-0232ade9-fa7c-4a5d-a08d-f20007037a24 - - - - -] Running command (rootwrap daemon): ['ip', 'link', 'set', 'tap25b2c0ed-0a', 'address', 'fa:16:3e:25:03:7b'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 17:40:26.642 5000 DEBUG neutron.agent.linux.utils [req-0232ade9-fa7c-4a5d-a08d-f20007037a24 - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 17:40:26.654 5000 DEBUG neutron.agent.linux.utils [req-0232ade9-fa7c-4a5d-a08d-f20007037a24 - - - - -] Running command (rootwrap daemon): ['ip', 'link', 'set', 'tap25b2c0ed-0a', 'netns', 'qdhcp-a3127656-d5cf-4734-b868-3ae134f10b1f'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 17:40:26.680 5000 DEBUG neutron.agent.linux.utils [req-0232ade9-fa7c-4a5d-a08d-f20007037a24 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-a3127656-d5cf-4734-b868-3ae134f10b1f', 'ip', 'link', 'set', 'tap25b2c0ed-0a', 'mtu', '1450'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 17:40:26.696 5000 DEBUG neutron.agent.linux.utils [req-0232ade9-fa7c-4a5d-a08d-f20007037a24 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-a3127656-d5cf-4734-b868-3ae134f10b1f', 'ip', 'link', 'set', 'tap25b2c0ed-0a', 'up'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 17:40:26.714 5000 DEBUG oslo_concurrency.lockutils [req-0232ade9-fa7c-4a5d-a08d-f20007037a24 - - - - -] Acquired semaphore "iptables-qdhcp-a3127656-d5cf-4734-b868-3ae134f10b1f" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 17:40:26.728 5000 DEBUG neutron.agent.linux.utils [req-0232ade9-fa7c-4a5d-a08d-f20007037a24 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-a3127656-d5cf-4734-b868-3ae134f10b1f', 'iptables-save'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 17:40:26.758 5000 DEBUG neutron.agent.linux.utils [req-0232ade9-fa7c-4a5d-a08d-f20007037a24 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-a3127656-d5cf-4734-b868-3ae134f10b1f', 'iptables-restore', '-n'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 17:40:26.781 5000 DEBUG neutron.agent.linux.iptables_manager [req-0232ade9-fa7c-4a5d-a08d-f20007037a24 - - - - -] IPTablesManager.apply completed with success. 56 iptables commands were issued _apply_synchronized /usr/lib/python2.7/site-packages/neutron/agent/linux/iptables_manager.py:601
2018-09-29 17:40:26.783 5000 DEBUG oslo_concurrency.lockutils [req-0232ade9-fa7c-4a5d-a08d-f20007037a24 - - - - -] Releasing semaphore "iptables-qdhcp-a3127656-d5cf-4734-b868-3ae134f10b1f" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 17:40:26.785 5000 DEBUG neutron.agent.linux.utils [req-0232ade9-fa7c-4a5d-a08d-f20007037a24 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-a3127656-d5cf-4734-b868-3ae134f10b1f', 'ip', '-o', 'link', 'show', 'tap25b2c0ed-0a'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 17:40:26.802 5000 DEBUG neutron.agent.linux.utils [req-0232ade9-fa7c-4a5d-a08d-f20007037a24 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-a3127656-d5cf-4734-b868-3ae134f10b1f', 'ip', 'addr', 'show', 'tap25b2c0ed-0a'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 17:40:26.819 5000 DEBUG neutron.agent.linux.utils [req-0232ade9-fa7c-4a5d-a08d-f20007037a24 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-a3127656-d5cf-4734-b868-3ae134f10b1f', 'ip', '-4', 'addr', 'add', '11.98.1.2/24', 'scope', 'global', 'dev', 'tap25b2c0ed-0a', 'brd', '11.98.1.255'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 17:40:26.835 5000 DEBUG neutron.agent.linux.utils [req-0232ade9-fa7c-4a5d-a08d-f20007037a24 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-a3127656-d5cf-4734-b868-3ae134f10b1f', 'ip', '-4', 'route', 'list', 'dev', 'tap25b2c0ed-0a'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 17:40:26.850 5000 DEBUG neutron.agent.linux.dhcp [req-0232ade9-fa7c-4a5d-a08d-f20007037a24 - - - - -] Setting IPv4 gateway for dhcp netns on net a3127656-d5cf-4734-b868-3ae134f10b1f to 11.98.1.1 _set_default_route_ip_version /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1169
2018-09-29 17:40:26.852 5000 DEBUG neutron.agent.linux.utils [req-0232ade9-fa7c-4a5d-a08d-f20007037a24 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-a3127656-d5cf-4734-b868-3ae134f10b1f', 'ip', '-4', 'route', 'replace', 'default', 'via', '11.98.1.1', 'dev', 'tap25b2c0ed-0a'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 17:40:26.870 5000 DEBUG neutron.agent.linux.utils [req-0232ade9-fa7c-4a5d-a08d-f20007037a24 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-a3127656-d5cf-4734-b868-3ae134f10b1f', 'ip', '-6', 'route', 'list', 'dev', 'tap25b2c0ed-0a'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 17:40:26.884 5000 DEBUG neutron.agent.linux.dhcp [req-0232ade9-fa7c-4a5d-a08d-f20007037a24 - - - - -] Cleaning stale devices for network a3127656-d5cf-4734-b868-3ae134f10b1f _cleanup_stale_devices /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1374
2018-09-29 17:40:26.885 5000 DEBUG neutron.agent.linux.utils [req-0232ade9-fa7c-4a5d-a08d-f20007037a24 - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 17:40:26.896 5000 DEBUG neutron.agent.linux.utils [req-0232ade9-fa7c-4a5d-a08d-f20007037a24 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-a3127656-d5cf-4734-b868-3ae134f10b1f', 'find', '/sys/class/net', '-maxdepth', '1', '-type', 'l', '-printf', '%f '] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 17:40:26.916 5000 DEBUG neutron.agent.linux.dhcp [req-0232ade9-fa7c-4a5d-a08d-f20007037a24 - - - - -] Building initial lease file: /var/lib/neutron/dhcp/a3127656-d5cf-4734-b868-3ae134f10b1f/leases _output_init_lease_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:626
2018-09-29 17:40:26.920 5000 DEBUG neutron.agent.linux.dhcp [req-0232ade9-fa7c-4a5d-a08d-f20007037a24 - - - - -] Done building initial lease file /var/lib/neutron/dhcp/a3127656-d5cf-4734-b868-3ae134f10b1f/leases with contents:
1538343626 fa:16:3e:25:03:7b 11.98.1.2 * *
 _output_init_lease_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:654
2018-09-29 17:40:26.921 5000 DEBUG neutron.agent.linux.dhcp [req-0232ade9-fa7c-4a5d-a08d-f20007037a24 - - - - -] Building host file: /var/lib/neutron/dhcp/a3127656-d5cf-4734-b868-3ae134f10b1f/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-29 17:40:26.923 5000 DEBUG neutron.agent.linux.dhcp [req-0232ade9-fa7c-4a5d-a08d-f20007037a24 - - - - -] Done building host file /var/lib/neutron/dhcp/a3127656-d5cf-4734-b868-3ae134f10b1f/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-29 17:40:26.927 5000 DEBUG neutron.agent.linux.utils [req-0232ade9-fa7c-4a5d-a08d-f20007037a24 - - - - -] Unable to access /var/lib/neutron/dhcp/a3127656-d5cf-4734-b868-3ae134f10b1f/pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 17:40:26.929 5000 DEBUG neutron.agent.linux.utils [req-0232ade9-fa7c-4a5d-a08d-f20007037a24 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-a3127656-d5cf-4734-b868-3ae134f10b1f', 'dnsmasq', '--no-hosts', '--no-resolv', '--strict-order', '--except-interface=lo', '--pid-file=/var/lib/neutron/dhcp/a3127656-d5cf-4734-b868-3ae134f10b1f/pid', '--dhcp-hostsfile=/var/lib/neutron/dhcp/a3127656-d5cf-4734-b868-3ae134f10b1f/host', '--addn-hosts=/var/lib/neutron/dhcp/a3127656-d5cf-4734-b868-3ae134f10b1f/addn_hosts', '--dhcp-optsfile=/var/lib/neutron/dhcp/a3127656-d5cf-4734-b868-3ae134f10b1f/opts', '--dhcp-leasefile=/var/lib/neutron/dhcp/a3127656-d5cf-4734-b868-3ae134f10b1f/leases', '--dhcp-match=set:ipxe,175', '--bind-interfaces', '--interface=tap25b2c0ed-0a', '--dhcp-range=set:tag0,11.98.1.0,static,255.255.255.0,86400s', '--dhcp-option-force=option:mtu,1450', '--dhcp-lease-max=256', '--conf-file=', '--domain=openstacklocal'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 17:40:26.970 5000 DEBUG neutron.agent.linux.utils [req-0232ade9-fa7c-4a5d-a08d-f20007037a24 - - - - -] Unable to access /var/lib/neutron/external/pids/a3127656-d5cf-4734-b868-3ae134f10b1f.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 17:40:26.972 5000 DEBUG neutron.agent.linux.utils [req-0232ade9-fa7c-4a5d-a08d-f20007037a24 - - - - -] Unable to access /var/lib/neutron/external/pids/a3127656-d5cf-4734-b868-3ae134f10b1f.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 17:40:26.974 5000 DEBUG neutron.agent.linux.external_process [req-0232ade9-fa7c-4a5d-a08d-f20007037a24 - - - - -] No process started for a3127656-d5cf-4734-b868-3ae134f10b1f disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-29 17:40:26.976 5000 DEBUG oslo_concurrency.lockutils [req-0232ade9-fa7c-4a5d-a08d-f20007037a24 - - - - -] Releasing semaphore "dhcp-agent-network-lock-a3127656-d5cf-4734-b868-3ae134f10b1f" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 17:40:26.978 5000 DEBUG oslo_concurrency.lockutils [req-06882cfd-0257-4eff-b3ed-d3084c938506 8fbe107dce56405f89cba07bca41fa33 d0066f73b1db4dcabfc61318ae559960 - - -] Acquired semaphore "dhcp-agent-network-lock-a3127656-d5cf-4734-b868-3ae134f10b1f" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 17:40:27.603 5000 DEBUG neutron.agent.dhcp.agent [req-e3818d13-1fde-4a26-9324-a9c00533d087 - - - - -] Calling driver for network: a3127656-d5cf-4734-b868-3ae134f10b1f action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-29 17:40:27.606 5000 DEBUG neutron.agent.linux.dhcp [req-e3818d13-1fde-4a26-9324-a9c00533d087 - - - - -] Building host file: /var/lib/neutron/dhcp/a3127656-d5cf-4734-b868-3ae134f10b1f/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-29 17:40:27.612 5000 DEBUG neutron.agent.linux.dhcp [req-e3818d13-1fde-4a26-9324-a9c00533d087 - - - - -] Done building host file /var/lib/neutron/dhcp/a3127656-d5cf-4734-b868-3ae134f10b1f/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-29 17:40:27.621 5000 DEBUG neutron.agent.linux.utils [req-e3818d13-1fde-4a26-9324-a9c00533d087 - - - - -] Running command (rootwrap daemon): ['kill', '-HUP', '9139'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 17:40:27.638 5000 DEBUG neutron.agent.linux.dhcp [req-e3818d13-1fde-4a26-9324-a9c00533d087 - - - - -] Reloading allocations for network: a3127656-d5cf-4734-b868-3ae134f10b1f reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:514
2018-09-29 17:40:27.640 5000 DEBUG neutron.agent.linux.utils [req-e3818d13-1fde-4a26-9324-a9c00533d087 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-a3127656-d5cf-4734-b868-3ae134f10b1f', 'ip', '-4', 'route', 'list', 'dev', 'tap25b2c0ed-0a'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 17:40:27.659 5000 DEBUG neutron.agent.linux.utils [req-e3818d13-1fde-4a26-9324-a9c00533d087 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-a3127656-d5cf-4734-b868-3ae134f10b1f', 'ip', '-6', 'route', 'list', 'dev', 'tap25b2c0ed-0a'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 17:40:27.681 5000 DEBUG neutron.agent.linux.utils [req-e3818d13-1fde-4a26-9324-a9c00533d087 - - - - -] Unable to access /var/lib/neutron/external/pids/a3127656-d5cf-4734-b868-3ae134f10b1f.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 17:40:27.682 5000 DEBUG neutron.agent.linux.utils [req-e3818d13-1fde-4a26-9324-a9c00533d087 - - - - -] Unable to access /var/lib/neutron/external/pids/a3127656-d5cf-4734-b868-3ae134f10b1f.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 17:40:27.683 5000 DEBUG neutron.agent.linux.external_process [req-e3818d13-1fde-4a26-9324-a9c00533d087 - - - - -] No process started for a3127656-d5cf-4734-b868-3ae134f10b1f disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-29 17:40:27.685 5000 DEBUG oslo_concurrency.lockutils [req-e3818d13-1fde-4a26-9324-a9c00533d087 - - - - -] Releasing semaphore "dhcp-agent-network-lock-a3127656-d5cf-4734-b868-3ae134f10b1f" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 17:40:27.687 5000 DEBUG oslo_concurrency.lockutils [req-0232ade9-fa7c-4a5d-a08d-f20007037a24 - - - - -] Acquired semaphore "dhcp-agent-network-lock-a3127656-d5cf-4734-b868-3ae134f10b1f" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 17:40:27.688 5000 DEBUG neutron.agent.dhcp.agent [req-0232ade9-fa7c-4a5d-a08d-f20007037a24 - - - - -] Discarding stale port update: admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-09-29T21:40:23Z, description=, device_id=dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-a3127656-d5cf-4734-b868-3ae134f10b1f, device_owner=network:dhcp, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'64865b83-e346-46f5-9979-be655b3faf5f', u'ip_address': u'11.98.1.2'}], id=25b2c0ed-0a35-4108-b73e-50a5ab73a807, mac_address=fa:16:3e:25:03:7b, name=, network_id=a3127656-d5cf-4734-b868-3ae134f10b1f, port_security_enabled=False, project_id=d0066f73b1db4dcabfc61318ae559960, revision_number=3, security_groups=[], status=DOWN, tags=[], tenant_id=d0066f73b1db4dcabfc61318ae559960, updated_at=2018-09-29T21:40:24Z port_update_end /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:410
2018-09-29 17:40:27.689 5000 DEBUG oslo_concurrency.lockutils [req-0232ade9-fa7c-4a5d-a08d-f20007037a24 - - - - -] Releasing semaphore "dhcp-agent-network-lock-a3127656-d5cf-4734-b868-3ae134f10b1f" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 17:40:27.690 5000 DEBUG oslo_concurrency.lockutils [req-0232ade9-fa7c-4a5d-a08d-f20007037a24 - - - - -] Acquired semaphore "dhcp-agent-network-lock-a3127656-d5cf-4734-b868-3ae134f10b1f" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 17:40:27.693 5000 INFO neutron.agent.dhcp.agent [req-0232ade9-fa7c-4a5d-a08d-f20007037a24 - - - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=datapath_type=system, ovs_hybrid_plug=True, port_filter=True, binding:vif_type=ovs, binding:vnic_type=normal, created_at=2018-09-29T21:40:23Z, description=, device_id=dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-a3127656-d5cf-4734-b868-3ae134f10b1f, device_owner=network:dhcp, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'64865b83-e346-46f5-9979-be655b3faf5f', u'ip_address': u'11.98.1.2'}], id=25b2c0ed-0a35-4108-b73e-50a5ab73a807, mac_address=fa:16:3e:25:03:7b, name=, network_id=a3127656-d5cf-4734-b868-3ae134f10b1f, port_security_enabled=False, project_id=d0066f73b1db4dcabfc61318ae559960, revision_number=4, security_groups=[], status=DOWN, tags=[], tenant_id=d0066f73b1db4dcabfc61318ae559960, updated_at=2018-09-29T21:40:25Z
2018-09-29 17:40:27.696 5000 DEBUG neutron.agent.dhcp.agent [req-0232ade9-fa7c-4a5d-a08d-f20007037a24 - - - - -] Calling driver for network: a3127656-d5cf-4734-b868-3ae134f10b1f action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-29 17:40:27.698 5000 DEBUG neutron.agent.linux.dhcp [req-0232ade9-fa7c-4a5d-a08d-f20007037a24 - - - - -] Building host file: /var/lib/neutron/dhcp/a3127656-d5cf-4734-b868-3ae134f10b1f/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-29 17:40:27.701 5000 DEBUG neutron.agent.linux.dhcp [req-0232ade9-fa7c-4a5d-a08d-f20007037a24 - - - - -] Done building host file /var/lib/neutron/dhcp/a3127656-d5cf-4734-b868-3ae134f10b1f/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-29 17:40:27.708 5000 DEBUG neutron.agent.linux.utils [req-0232ade9-fa7c-4a5d-a08d-f20007037a24 - - - - -] Running command (rootwrap daemon): ['kill', '-HUP', '9139'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 17:40:27.730 5000 DEBUG neutron.agent.linux.dhcp [req-0232ade9-fa7c-4a5d-a08d-f20007037a24 - - - - -] Reloading allocations for network: a3127656-d5cf-4734-b868-3ae134f10b1f reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:514
2018-09-29 17:40:27.732 5000 DEBUG neutron.agent.linux.utils [req-0232ade9-fa7c-4a5d-a08d-f20007037a24 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-a3127656-d5cf-4734-b868-3ae134f10b1f', 'ip', '-4', 'route', 'list', 'dev', 'tap25b2c0ed-0a'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 17:40:27.751 5000 DEBUG neutron.agent.linux.utils [req-0232ade9-fa7c-4a5d-a08d-f20007037a24 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-a3127656-d5cf-4734-b868-3ae134f10b1f', 'ip', '-6', 'route', 'list', 'dev', 'tap25b2c0ed-0a'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 17:40:27.773 5000 DEBUG neutron.agent.linux.utils [req-0232ade9-fa7c-4a5d-a08d-f20007037a24 - - - - -] Unable to access /var/lib/neutron/external/pids/a3127656-d5cf-4734-b868-3ae134f10b1f.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 17:40:27.775 5000 DEBUG neutron.agent.linux.utils [req-0232ade9-fa7c-4a5d-a08d-f20007037a24 - - - - -] Unable to access /var/lib/neutron/external/pids/a3127656-d5cf-4734-b868-3ae134f10b1f.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 17:40:27.775 5000 DEBUG neutron.agent.linux.external_process [req-0232ade9-fa7c-4a5d-a08d-f20007037a24 - - - - -] No process started for a3127656-d5cf-4734-b868-3ae134f10b1f disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-29 17:40:27.776 5000 DEBUG oslo_concurrency.lockutils [req-0232ade9-fa7c-4a5d-a08d-f20007037a24 - - - - -] Releasing semaphore "dhcp-agent-network-lock-a3127656-d5cf-4734-b868-3ae134f10b1f" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 17:40:35.446 5000 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-29 17:40:35.450 5000 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.004s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-29 17:41:05.065 5000 DEBUG oslo_concurrency.lockutils [req-fb9943a2-e1db-4b46-afaa-44152a7deb18 8fbe107dce56405f89cba07bca41fa33 d0066f73b1db4dcabfc61318ae559960 - - -] Acquired semaphore "dhcp-agent-network-lock-a3127656-d5cf-4734-b868-3ae134f10b1f" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 17:41:05.066 5000 INFO neutron.agent.dhcp.agent [req-fb9943a2-e1db-4b46-afaa-44152a7deb18 8fbe107dce56405f89cba07bca41fa33 d0066f73b1db4dcabfc61318ae559960 - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-09-29T21:41:04Z, description=, device_id=07a70ff8-54a1-4c30-b713-d0037256ab2c, device_owner=network:router_interface, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'64865b83-e346-46f5-9979-be655b3faf5f', u'ip_address': u'11.98.1.1'}], id=cf9837c1-af07-4395-b254-38152027ca7c, mac_address=fa:16:3e:5b:36:b6, name=, network_id=a3127656-d5cf-4734-b868-3ae134f10b1f, port_security_enabled=False, project_id=d0066f73b1db4dcabfc61318ae559960, revision_number=3, security_groups=[], status=DOWN, tags=[], tenant_id=d0066f73b1db4dcabfc61318ae559960, updated_at=2018-09-29T21:41:04Z
2018-09-29 17:41:05.068 5000 DEBUG neutron.agent.dhcp.agent [req-fb9943a2-e1db-4b46-afaa-44152a7deb18 8fbe107dce56405f89cba07bca41fa33 d0066f73b1db4dcabfc61318ae559960 - - -] Calling driver for network: a3127656-d5cf-4734-b868-3ae134f10b1f action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-29 17:41:05.070 5000 DEBUG neutron.agent.linux.dhcp [req-fb9943a2-e1db-4b46-afaa-44152a7deb18 8fbe107dce56405f89cba07bca41fa33 d0066f73b1db4dcabfc61318ae559960 - - -] Building host file: /var/lib/neutron/dhcp/a3127656-d5cf-4734-b868-3ae134f10b1f/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-29 17:41:05.073 5000 DEBUG neutron.agent.linux.dhcp [req-fb9943a2-e1db-4b46-afaa-44152a7deb18 8fbe107dce56405f89cba07bca41fa33 d0066f73b1db4dcabfc61318ae559960 - - -] Done building host file /var/lib/neutron/dhcp/a3127656-d5cf-4734-b868-3ae134f10b1f/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-29 17:41:05.079 5000 DEBUG neutron.agent.linux.utils [req-fb9943a2-e1db-4b46-afaa-44152a7deb18 8fbe107dce56405f89cba07bca41fa33 d0066f73b1db4dcabfc61318ae559960 - - -] Running command (rootwrap daemon): ['kill', '-HUP', '9139'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 17:41:05.104 5000 DEBUG neutron.agent.linux.dhcp [req-fb9943a2-e1db-4b46-afaa-44152a7deb18 8fbe107dce56405f89cba07bca41fa33 d0066f73b1db4dcabfc61318ae559960 - - -] Reloading allocations for network: a3127656-d5cf-4734-b868-3ae134f10b1f reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:514
2018-09-29 17:41:05.106 5000 DEBUG neutron.agent.linux.utils [req-fb9943a2-e1db-4b46-afaa-44152a7deb18 8fbe107dce56405f89cba07bca41fa33 d0066f73b1db4dcabfc61318ae559960 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-a3127656-d5cf-4734-b868-3ae134f10b1f', 'ip', '-4', 'route', 'list', 'dev', 'tap25b2c0ed-0a'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 17:41:05.124 5000 DEBUG neutron.agent.linux.utils [req-fb9943a2-e1db-4b46-afaa-44152a7deb18 8fbe107dce56405f89cba07bca41fa33 d0066f73b1db4dcabfc61318ae559960 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-a3127656-d5cf-4734-b868-3ae134f10b1f', 'ip', '-6', 'route', 'list', 'dev', 'tap25b2c0ed-0a'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 17:41:05.140 5000 DEBUG neutron.agent.linux.utils [req-fb9943a2-e1db-4b46-afaa-44152a7deb18 8fbe107dce56405f89cba07bca41fa33 d0066f73b1db4dcabfc61318ae559960 - - -] Unable to access /var/lib/neutron/external/pids/a3127656-d5cf-4734-b868-3ae134f10b1f.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 17:41:05.142 5000 DEBUG neutron.agent.linux.utils [req-fb9943a2-e1db-4b46-afaa-44152a7deb18 8fbe107dce56405f89cba07bca41fa33 d0066f73b1db4dcabfc61318ae559960 - - -] Unable to access /var/lib/neutron/external/pids/a3127656-d5cf-4734-b868-3ae134f10b1f.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 17:41:05.142 5000 DEBUG neutron.agent.linux.external_process [req-fb9943a2-e1db-4b46-afaa-44152a7deb18 8fbe107dce56405f89cba07bca41fa33 d0066f73b1db4dcabfc61318ae559960 - - -] No process started for a3127656-d5cf-4734-b868-3ae134f10b1f disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-29 17:41:05.143 5000 DEBUG oslo_concurrency.lockutils [req-fb9943a2-e1db-4b46-afaa-44152a7deb18 8fbe107dce56405f89cba07bca41fa33 d0066f73b1db4dcabfc61318ae559960 - - -] Releasing semaphore "dhcp-agent-network-lock-a3127656-d5cf-4734-b868-3ae134f10b1f" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 17:41:06.459 5000 DEBUG oslo_concurrency.lockutils [req-fb9943a2-e1db-4b46-afaa-44152a7deb18 8fbe107dce56405f89cba07bca41fa33 d0066f73b1db4dcabfc61318ae559960 - - -] Acquired semaphore "dhcp-agent-network-lock-a3127656-d5cf-4734-b868-3ae134f10b1f" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 17:41:06.460 5000 INFO neutron.agent.dhcp.agent [req-fb9943a2-e1db-4b46-afaa-44152a7deb18 8fbe107dce56405f89cba07bca41fa33 d0066f73b1db4dcabfc61318ae559960 - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-09-29T21:41:04Z, description=, device_id=07a70ff8-54a1-4c30-b713-d0037256ab2c, device_owner=network:router_interface, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'64865b83-e346-46f5-9979-be655b3faf5f', u'ip_address': u'11.98.1.1'}], id=cf9837c1-af07-4395-b254-38152027ca7c, mac_address=fa:16:3e:5b:36:b6, name=, network_id=a3127656-d5cf-4734-b868-3ae134f10b1f, port_security_enabled=False, project_id=d0066f73b1db4dcabfc61318ae559960, revision_number=3, security_groups=[], status=DOWN, tags=[], tenant_id=d0066f73b1db4dcabfc61318ae559960, updated_at=2018-09-29T21:41:04Z
2018-09-29 17:41:06.462 5000 DEBUG neutron.agent.dhcp.agent [req-fb9943a2-e1db-4b46-afaa-44152a7deb18 8fbe107dce56405f89cba07bca41fa33 d0066f73b1db4dcabfc61318ae559960 - - -] Calling driver for network: a3127656-d5cf-4734-b868-3ae134f10b1f action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-29 17:41:06.464 5000 DEBUG neutron.agent.linux.dhcp [req-fb9943a2-e1db-4b46-afaa-44152a7deb18 8fbe107dce56405f89cba07bca41fa33 d0066f73b1db4dcabfc61318ae559960 - - -] Building host file: /var/lib/neutron/dhcp/a3127656-d5cf-4734-b868-3ae134f10b1f/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-29 17:41:06.467 5000 DEBUG neutron.agent.linux.dhcp [req-fb9943a2-e1db-4b46-afaa-44152a7deb18 8fbe107dce56405f89cba07bca41fa33 d0066f73b1db4dcabfc61318ae559960 - - -] Done building host file /var/lib/neutron/dhcp/a3127656-d5cf-4734-b868-3ae134f10b1f/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-29 17:41:06.473 5000 DEBUG neutron.agent.linux.utils [req-fb9943a2-e1db-4b46-afaa-44152a7deb18 8fbe107dce56405f89cba07bca41fa33 d0066f73b1db4dcabfc61318ae559960 - - -] Running command (rootwrap daemon): ['kill', '-HUP', '9139'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 17:41:06.490 5000 DEBUG neutron.agent.linux.dhcp [req-fb9943a2-e1db-4b46-afaa-44152a7deb18 8fbe107dce56405f89cba07bca41fa33 d0066f73b1db4dcabfc61318ae559960 - - -] Reloading allocations for network: a3127656-d5cf-4734-b868-3ae134f10b1f reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:514
2018-09-29 17:41:06.493 5000 DEBUG neutron.agent.linux.utils [req-fb9943a2-e1db-4b46-afaa-44152a7deb18 8fbe107dce56405f89cba07bca41fa33 d0066f73b1db4dcabfc61318ae559960 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-a3127656-d5cf-4734-b868-3ae134f10b1f', 'ip', '-4', 'route', 'list', 'dev', 'tap25b2c0ed-0a'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 17:41:06.506 5000 DEBUG neutron.agent.linux.utils [req-fb9943a2-e1db-4b46-afaa-44152a7deb18 8fbe107dce56405f89cba07bca41fa33 d0066f73b1db4dcabfc61318ae559960 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-a3127656-d5cf-4734-b868-3ae134f10b1f', 'ip', '-6', 'route', 'list', 'dev', 'tap25b2c0ed-0a'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 17:41:06.522 5000 DEBUG neutron.agent.linux.utils [req-fb9943a2-e1db-4b46-afaa-44152a7deb18 8fbe107dce56405f89cba07bca41fa33 d0066f73b1db4dcabfc61318ae559960 - - -] Unable to access /var/lib/neutron/external/pids/a3127656-d5cf-4734-b868-3ae134f10b1f.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 17:41:06.523 5000 DEBUG neutron.agent.linux.utils [req-fb9943a2-e1db-4b46-afaa-44152a7deb18 8fbe107dce56405f89cba07bca41fa33 d0066f73b1db4dcabfc61318ae559960 - - -] Unable to access /var/lib/neutron/external/pids/a3127656-d5cf-4734-b868-3ae134f10b1f.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 17:41:06.524 5000 DEBUG neutron.agent.linux.external_process [req-fb9943a2-e1db-4b46-afaa-44152a7deb18 8fbe107dce56405f89cba07bca41fa33 d0066f73b1db4dcabfc61318ae559960 - - -] No process started for a3127656-d5cf-4734-b868-3ae134f10b1f disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-29 17:41:06.524 5000 DEBUG oslo_concurrency.lockutils [req-fb9943a2-e1db-4b46-afaa-44152a7deb18 8fbe107dce56405f89cba07bca41fa33 d0066f73b1db4dcabfc61318ae559960 - - -] Releasing semaphore "dhcp-agent-network-lock-a3127656-d5cf-4734-b868-3ae134f10b1f" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 17:41:09.313 5000 DEBUG oslo_concurrency.lockutils [req-72c24686-b36f-465a-9546-452a1fc7cb2a - - - - -] Acquired semaphore "dhcp-agent-network-lock-a3127656-d5cf-4734-b868-3ae134f10b1f" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 17:41:09.315 5000 INFO neutron.agent.dhcp.agent [req-72c24686-b36f-465a-9546-452a1fc7cb2a - - - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-09-29T21:41:04Z, description=, device_id=07a70ff8-54a1-4c30-b713-d0037256ab2c, device_owner=network:router_interface, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'64865b83-e346-46f5-9979-be655b3faf5f', u'ip_address': u'11.98.1.1'}], id=cf9837c1-af07-4395-b254-38152027ca7c, mac_address=fa:16:3e:5b:36:b6, name=, network_id=a3127656-d5cf-4734-b868-3ae134f10b1f, port_security_enabled=False, project_id=d0066f73b1db4dcabfc61318ae559960, revision_number=4, security_groups=[], status=DOWN, tags=[], tenant_id=d0066f73b1db4dcabfc61318ae559960, updated_at=2018-09-29T21:41:05Z
2018-09-29 17:41:09.316 5000 DEBUG neutron.agent.dhcp.agent [req-72c24686-b36f-465a-9546-452a1fc7cb2a - - - - -] Calling driver for network: a3127656-d5cf-4734-b868-3ae134f10b1f action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-29 17:41:09.319 5000 DEBUG neutron.agent.linux.dhcp [req-72c24686-b36f-465a-9546-452a1fc7cb2a - - - - -] Building host file: /var/lib/neutron/dhcp/a3127656-d5cf-4734-b868-3ae134f10b1f/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-29 17:41:09.321 5000 DEBUG neutron.agent.linux.dhcp [req-72c24686-b36f-465a-9546-452a1fc7cb2a - - - - -] Done building host file /var/lib/neutron/dhcp/a3127656-d5cf-4734-b868-3ae134f10b1f/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-29 17:41:09.325 5000 DEBUG neutron.agent.linux.utils [req-72c24686-b36f-465a-9546-452a1fc7cb2a - - - - -] Running command (rootwrap daemon): ['kill', '-HUP', '9139'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 17:41:09.345 5000 DEBUG neutron.agent.linux.dhcp [req-72c24686-b36f-465a-9546-452a1fc7cb2a - - - - -] Reloading allocations for network: a3127656-d5cf-4734-b868-3ae134f10b1f reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:514
2018-09-29 17:41:09.347 5000 DEBUG neutron.agent.linux.utils [req-72c24686-b36f-465a-9546-452a1fc7cb2a - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-a3127656-d5cf-4734-b868-3ae134f10b1f', 'ip', '-4', 'route', 'list', 'dev', 'tap25b2c0ed-0a'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 17:41:09.366 5000 DEBUG neutron.agent.linux.utils [req-72c24686-b36f-465a-9546-452a1fc7cb2a - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-a3127656-d5cf-4734-b868-3ae134f10b1f', 'ip', '-6', 'route', 'list', 'dev', 'tap25b2c0ed-0a'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 17:41:09.381 5000 DEBUG neutron.agent.linux.utils [req-72c24686-b36f-465a-9546-452a1fc7cb2a - - - - -] Unable to access /var/lib/neutron/external/pids/a3127656-d5cf-4734-b868-3ae134f10b1f.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 17:41:09.382 5000 DEBUG neutron.agent.linux.utils [req-72c24686-b36f-465a-9546-452a1fc7cb2a - - - - -] Unable to access /var/lib/neutron/external/pids/a3127656-d5cf-4734-b868-3ae134f10b1f.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 17:41:09.384 5000 DEBUG neutron.agent.linux.external_process [req-72c24686-b36f-465a-9546-452a1fc7cb2a - - - - -] No process started for a3127656-d5cf-4734-b868-3ae134f10b1f disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-29 17:41:09.384 5000 DEBUG oslo_concurrency.lockutils [req-72c24686-b36f-465a-9546-452a1fc7cb2a - - - - -] Releasing semaphore "dhcp-agent-network-lock-a3127656-d5cf-4734-b868-3ae134f10b1f" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 17:41:09.977 5000 DEBUG oslo_concurrency.lockutils [req-72c24686-b36f-465a-9546-452a1fc7cb2a - - - - -] Acquired semaphore "dhcp-agent-network-lock-a3127656-d5cf-4734-b868-3ae134f10b1f" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 17:41:09.978 5000 INFO neutron.agent.dhcp.agent [req-72c24686-b36f-465a-9546-452a1fc7cb2a - - - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=datapath_type=system, ovs_hybrid_plug=True, port_filter=True, binding:vif_type=ovs, binding:vnic_type=normal, created_at=2018-09-29T21:41:04Z, description=, device_id=07a70ff8-54a1-4c30-b713-d0037256ab2c, device_owner=network:router_interface, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'64865b83-e346-46f5-9979-be655b3faf5f', u'ip_address': u'11.98.1.1'}], id=cf9837c1-af07-4395-b254-38152027ca7c, mac_address=fa:16:3e:5b:36:b6, name=, network_id=a3127656-d5cf-4734-b868-3ae134f10b1f, port_security_enabled=False, project_id=d0066f73b1db4dcabfc61318ae559960, revision_number=6, security_groups=[], status=DOWN, tags=[], tenant_id=d0066f73b1db4dcabfc61318ae559960, updated_at=2018-09-29T21:41:09Z
2018-09-29 17:41:09.980 5000 DEBUG neutron.agent.dhcp.agent [req-72c24686-b36f-465a-9546-452a1fc7cb2a - - - - -] Calling driver for network: a3127656-d5cf-4734-b868-3ae134f10b1f action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-29 17:41:09.981 5000 DEBUG neutron.agent.linux.dhcp [req-72c24686-b36f-465a-9546-452a1fc7cb2a - - - - -] Building host file: /var/lib/neutron/dhcp/a3127656-d5cf-4734-b868-3ae134f10b1f/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-29 17:41:09.984 5000 DEBUG neutron.agent.linux.dhcp [req-72c24686-b36f-465a-9546-452a1fc7cb2a - - - - -] Done building host file /var/lib/neutron/dhcp/a3127656-d5cf-4734-b868-3ae134f10b1f/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-29 17:41:09.989 5000 DEBUG neutron.agent.linux.utils [req-72c24686-b36f-465a-9546-452a1fc7cb2a - - - - -] Running command (rootwrap daemon): ['kill', '-HUP', '9139'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 17:41:10.006 5000 DEBUG neutron.agent.linux.dhcp [req-72c24686-b36f-465a-9546-452a1fc7cb2a - - - - -] Reloading allocations for network: a3127656-d5cf-4734-b868-3ae134f10b1f reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:514
2018-09-29 17:41:10.008 5000 DEBUG neutron.agent.linux.utils [req-72c24686-b36f-465a-9546-452a1fc7cb2a - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-a3127656-d5cf-4734-b868-3ae134f10b1f', 'ip', '-4', 'route', 'list', 'dev', 'tap25b2c0ed-0a'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 17:41:10.023 5000 DEBUG neutron.agent.linux.utils [req-72c24686-b36f-465a-9546-452a1fc7cb2a - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-a3127656-d5cf-4734-b868-3ae134f10b1f', 'ip', '-6', 'route', 'list', 'dev', 'tap25b2c0ed-0a'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 17:41:10.040 5000 DEBUG neutron.agent.linux.utils [req-72c24686-b36f-465a-9546-452a1fc7cb2a - - - - -] Unable to access /var/lib/neutron/external/pids/a3127656-d5cf-4734-b868-3ae134f10b1f.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 17:41:10.041 5000 DEBUG neutron.agent.linux.utils [req-72c24686-b36f-465a-9546-452a1fc7cb2a - - - - -] Unable to access /var/lib/neutron/external/pids/a3127656-d5cf-4734-b868-3ae134f10b1f.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 17:41:10.042 5000 DEBUG neutron.agent.linux.external_process [req-72c24686-b36f-465a-9546-452a1fc7cb2a - - - - -] No process started for a3127656-d5cf-4734-b868-3ae134f10b1f disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-29 17:41:10.043 5000 DEBUG oslo_concurrency.lockutils [req-72c24686-b36f-465a-9546-452a1fc7cb2a - - - - -] Releasing semaphore "dhcp-agent-network-lock-a3127656-d5cf-4734-b868-3ae134f10b1f" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 17:41:19.488 5000 DEBUG oslo_concurrency.lockutils [req-fa7f1e4e-e6aa-4d4f-80c8-9379475235bb 8fbe107dce56405f89cba07bca41fa33 d0066f73b1db4dcabfc61318ae559960 - - -] Acquired semaphore "dhcp-agent-network-lock-40fff4bc-a326-419b-b1c1-a7aa26b2d41a" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 17:41:19.489 5000 DEBUG oslo_concurrency.lockutils [req-fa7f1e4e-e6aa-4d4f-80c8-9379475235bb 8fbe107dce56405f89cba07bca41fa33 d0066f73b1db4dcabfc61318ae559960 - - -] Releasing semaphore "dhcp-agent-network-lock-40fff4bc-a326-419b-b1c1-a7aa26b2d41a" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 17:41:23.093 5000 DEBUG oslo_concurrency.lockutils [req-bcefcc9b-f45e-4ff1-be68-7f350836ef39 - - - - -] Acquired semaphore "dhcp-agent-network-lock-40fff4bc-a326-419b-b1c1-a7aa26b2d41a" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 17:41:23.094 5000 DEBUG oslo_concurrency.lockutils [req-bcefcc9b-f45e-4ff1-be68-7f350836ef39 - - - - -] Releasing semaphore "dhcp-agent-network-lock-40fff4bc-a326-419b-b1c1-a7aa26b2d41a" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 17:41:24.482 5000 DEBUG oslo_concurrency.lockutils [req-bcefcc9b-f45e-4ff1-be68-7f350836ef39 - - - - -] Acquired semaphore "dhcp-agent-network-lock-40fff4bc-a326-419b-b1c1-a7aa26b2d41a" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 17:41:24.483 5000 DEBUG oslo_concurrency.lockutils [req-bcefcc9b-f45e-4ff1-be68-7f350836ef39 - - - - -] Releasing semaphore "dhcp-agent-network-lock-40fff4bc-a326-419b-b1c1-a7aa26b2d41a" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 17:41:35.446 5000 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-29 17:41:35.450 5000 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.004s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-29 17:41:53.777 5000 DEBUG oslo_concurrency.lockutils [req-af2f50dd-3e66-4bcb-8a96-3255bc6908ff 8fbe107dce56405f89cba07bca41fa33 d0066f73b1db4dcabfc61318ae559960 - - -] Acquired semaphore "dhcp-agent-network-lock-a3127656-d5cf-4734-b868-3ae134f10b1f" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 17:41:53.778 5000 INFO neutron.agent.dhcp.agent [req-af2f50dd-3e66-4bcb-8a96-3255bc6908ff 8fbe107dce56405f89cba07bca41fa33 d0066f73b1db4dcabfc61318ae559960 - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-09-29T21:41:51Z, description=, device_id=44a449d3-20d9-4493-831d-2f31c5cda54b, device_owner=, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'64865b83-e346-46f5-9979-be655b3faf5f', u'ip_address': u'11.98.1.3'}], id=c5f2ebd3-42fe-4e96-9bf8-053f0ebeabae, mac_address=fa:16:3e:24:70:8e, name=, network_id=a3127656-d5cf-4734-b868-3ae134f10b1f, port_security_enabled=True, project_id=d0066f73b1db4dcabfc61318ae559960, revision_number=3, security_groups=[u'ff61e9f5-8d44-4961-813f-e6c86fb239fe'], status=DOWN, tags=[], tenant_id=d0066f73b1db4dcabfc61318ae559960, updated_at=2018-09-29T21:41:52Z
2018-09-29 17:41:53.781 5000 DEBUG neutron.agent.dhcp.agent [req-af2f50dd-3e66-4bcb-8a96-3255bc6908ff 8fbe107dce56405f89cba07bca41fa33 d0066f73b1db4dcabfc61318ae559960 - - -] Calling driver for network: a3127656-d5cf-4734-b868-3ae134f10b1f action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-29 17:41:53.783 5000 DEBUG neutron.agent.linux.dhcp [req-af2f50dd-3e66-4bcb-8a96-3255bc6908ff 8fbe107dce56405f89cba07bca41fa33 d0066f73b1db4dcabfc61318ae559960 - - -] Building host file: /var/lib/neutron/dhcp/a3127656-d5cf-4734-b868-3ae134f10b1f/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-29 17:41:53.787 5000 DEBUG neutron.agent.linux.dhcp [req-af2f50dd-3e66-4bcb-8a96-3255bc6908ff 8fbe107dce56405f89cba07bca41fa33 d0066f73b1db4dcabfc61318ae559960 - - -] Done building host file /var/lib/neutron/dhcp/a3127656-d5cf-4734-b868-3ae134f10b1f/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-29 17:41:53.794 5000 DEBUG neutron.agent.linux.utils [req-af2f50dd-3e66-4bcb-8a96-3255bc6908ff 8fbe107dce56405f89cba07bca41fa33 d0066f73b1db4dcabfc61318ae559960 - - -] Running command (rootwrap daemon): ['kill', '-HUP', '9139'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 17:41:53.815 5000 DEBUG neutron.agent.linux.dhcp [req-af2f50dd-3e66-4bcb-8a96-3255bc6908ff 8fbe107dce56405f89cba07bca41fa33 d0066f73b1db4dcabfc61318ae559960 - - -] Reloading allocations for network: a3127656-d5cf-4734-b868-3ae134f10b1f reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:514
2018-09-29 17:41:53.817 5000 DEBUG neutron.agent.linux.utils [req-af2f50dd-3e66-4bcb-8a96-3255bc6908ff 8fbe107dce56405f89cba07bca41fa33 d0066f73b1db4dcabfc61318ae559960 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-a3127656-d5cf-4734-b868-3ae134f10b1f', 'ip', '-4', 'route', 'list', 'dev', 'tap25b2c0ed-0a'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 17:41:53.833 5000 DEBUG neutron.agent.linux.utils [req-af2f50dd-3e66-4bcb-8a96-3255bc6908ff 8fbe107dce56405f89cba07bca41fa33 d0066f73b1db4dcabfc61318ae559960 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-a3127656-d5cf-4734-b868-3ae134f10b1f', 'ip', '-6', 'route', 'list', 'dev', 'tap25b2c0ed-0a'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 17:41:53.849 5000 DEBUG neutron.agent.linux.utils [req-af2f50dd-3e66-4bcb-8a96-3255bc6908ff 8fbe107dce56405f89cba07bca41fa33 d0066f73b1db4dcabfc61318ae559960 - - -] Unable to access /var/lib/neutron/external/pids/a3127656-d5cf-4734-b868-3ae134f10b1f.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 17:41:53.851 5000 DEBUG neutron.agent.linux.utils [req-af2f50dd-3e66-4bcb-8a96-3255bc6908ff 8fbe107dce56405f89cba07bca41fa33 d0066f73b1db4dcabfc61318ae559960 - - -] Unable to access /var/lib/neutron/external/pids/a3127656-d5cf-4734-b868-3ae134f10b1f.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 17:41:53.852 5000 DEBUG neutron.agent.linux.external_process [req-af2f50dd-3e66-4bcb-8a96-3255bc6908ff 8fbe107dce56405f89cba07bca41fa33 d0066f73b1db4dcabfc61318ae559960 - - -] No process started for a3127656-d5cf-4734-b868-3ae134f10b1f disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-29 17:41:53.853 5000 DEBUG oslo_concurrency.lockutils [req-af2f50dd-3e66-4bcb-8a96-3255bc6908ff 8fbe107dce56405f89cba07bca41fa33 d0066f73b1db4dcabfc61318ae559960 - - -] Releasing semaphore "dhcp-agent-network-lock-a3127656-d5cf-4734-b868-3ae134f10b1f" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 17:41:57.608 5000 DEBUG oslo_concurrency.lockutils [req-5037b967-75f2-451b-be39-05e3e0c6fab3 6041b01c631d425d829cf96874ce50bf 671ea1986554476bbcaa620e00697d23 - - -] Acquired semaphore "dhcp-agent-network-lock-a3127656-d5cf-4734-b868-3ae134f10b1f" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 17:41:57.610 5000 INFO neutron.agent.dhcp.agent [req-5037b967-75f2-451b-be39-05e3e0c6fab3 6041b01c631d425d829cf96874ce50bf 671ea1986554476bbcaa620e00697d23 - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-09-29T21:41:51Z, description=, device_id=44a449d3-20d9-4493-831d-2f31c5cda54b, device_owner=compute:nova, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'64865b83-e346-46f5-9979-be655b3faf5f', u'ip_address': u'11.98.1.3'}], id=c5f2ebd3-42fe-4e96-9bf8-053f0ebeabae, mac_address=fa:16:3e:24:70:8e, name=, network_id=a3127656-d5cf-4734-b868-3ae134f10b1f, port_security_enabled=True, project_id=d0066f73b1db4dcabfc61318ae559960, revision_number=4, security_groups=[u'ff61e9f5-8d44-4961-813f-e6c86fb239fe'], status=DOWN, tags=[], tenant_id=d0066f73b1db4dcabfc61318ae559960, updated_at=2018-09-29T21:41:56Z
2018-09-29 17:41:57.613 5000 DEBUG neutron.agent.dhcp.agent [req-5037b967-75f2-451b-be39-05e3e0c6fab3 6041b01c631d425d829cf96874ce50bf 671ea1986554476bbcaa620e00697d23 - - -] Calling driver for network: a3127656-d5cf-4734-b868-3ae134f10b1f action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-29 17:41:57.616 5000 DEBUG neutron.agent.linux.dhcp [req-5037b967-75f2-451b-be39-05e3e0c6fab3 6041b01c631d425d829cf96874ce50bf 671ea1986554476bbcaa620e00697d23 - - -] Building host file: /var/lib/neutron/dhcp/a3127656-d5cf-4734-b868-3ae134f10b1f/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-29 17:41:57.620 5000 DEBUG neutron.agent.linux.dhcp [req-5037b967-75f2-451b-be39-05e3e0c6fab3 6041b01c631d425d829cf96874ce50bf 671ea1986554476bbcaa620e00697d23 - - -] Done building host file /var/lib/neutron/dhcp/a3127656-d5cf-4734-b868-3ae134f10b1f/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-29 17:41:57.628 5000 DEBUG neutron.agent.linux.utils [req-5037b967-75f2-451b-be39-05e3e0c6fab3 6041b01c631d425d829cf96874ce50bf 671ea1986554476bbcaa620e00697d23 - - -] Running command (rootwrap daemon): ['kill', '-HUP', '9139'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 17:41:57.648 5000 DEBUG neutron.agent.linux.dhcp [req-5037b967-75f2-451b-be39-05e3e0c6fab3 6041b01c631d425d829cf96874ce50bf 671ea1986554476bbcaa620e00697d23 - - -] Reloading allocations for network: a3127656-d5cf-4734-b868-3ae134f10b1f reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:514
2018-09-29 17:41:57.651 5000 DEBUG neutron.agent.linux.utils [req-5037b967-75f2-451b-be39-05e3e0c6fab3 6041b01c631d425d829cf96874ce50bf 671ea1986554476bbcaa620e00697d23 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-a3127656-d5cf-4734-b868-3ae134f10b1f', 'ip', '-4', 'route', 'list', 'dev', 'tap25b2c0ed-0a'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 17:41:57.675 5000 DEBUG neutron.agent.linux.utils [req-5037b967-75f2-451b-be39-05e3e0c6fab3 6041b01c631d425d829cf96874ce50bf 671ea1986554476bbcaa620e00697d23 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-a3127656-d5cf-4734-b868-3ae134f10b1f', 'ip', '-6', 'route', 'list', 'dev', 'tap25b2c0ed-0a'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 17:41:57.696 5000 DEBUG neutron.agent.linux.utils [req-5037b967-75f2-451b-be39-05e3e0c6fab3 6041b01c631d425d829cf96874ce50bf 671ea1986554476bbcaa620e00697d23 - - -] Unable to access /var/lib/neutron/external/pids/a3127656-d5cf-4734-b868-3ae134f10b1f.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 17:41:57.697 5000 DEBUG neutron.agent.linux.utils [req-5037b967-75f2-451b-be39-05e3e0c6fab3 6041b01c631d425d829cf96874ce50bf 671ea1986554476bbcaa620e00697d23 - - -] Unable to access /var/lib/neutron/external/pids/a3127656-d5cf-4734-b868-3ae134f10b1f.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 17:41:57.699 5000 DEBUG neutron.agent.linux.external_process [req-5037b967-75f2-451b-be39-05e3e0c6fab3 6041b01c631d425d829cf96874ce50bf 671ea1986554476bbcaa620e00697d23 - - -] No process started for a3127656-d5cf-4734-b868-3ae134f10b1f disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-29 17:41:57.701 5000 DEBUG oslo_concurrency.lockutils [req-5037b967-75f2-451b-be39-05e3e0c6fab3 6041b01c631d425d829cf96874ce50bf 671ea1986554476bbcaa620e00697d23 - - -] Releasing semaphore "dhcp-agent-network-lock-a3127656-d5cf-4734-b868-3ae134f10b1f" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 17:41:58.654 5000 DEBUG oslo_concurrency.lockutils [req-5037b967-75f2-451b-be39-05e3e0c6fab3 6041b01c631d425d829cf96874ce50bf 671ea1986554476bbcaa620e00697d23 - - -] Acquired semaphore "dhcp-agent-network-lock-a3127656-d5cf-4734-b868-3ae134f10b1f" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 17:41:58.656 5000 INFO neutron.agent.dhcp.agent [req-5037b967-75f2-451b-be39-05e3e0c6fab3 6041b01c631d425d829cf96874ce50bf 671ea1986554476bbcaa620e00697d23 - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=datapath_type=system, ovs_hybrid_plug=True, port_filter=True, binding:vif_type=ovs, binding:vnic_type=normal, created_at=2018-09-29T21:41:51Z, description=, device_id=44a449d3-20d9-4493-831d-2f31c5cda54b, device_owner=compute:nova, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'64865b83-e346-46f5-9979-be655b3faf5f', u'ip_address': u'11.98.1.3'}], id=c5f2ebd3-42fe-4e96-9bf8-053f0ebeabae, mac_address=fa:16:3e:24:70:8e, name=, network_id=a3127656-d5cf-4734-b868-3ae134f10b1f, port_security_enabled=True, project_id=d0066f73b1db4dcabfc61318ae559960, revision_number=5, security_groups=[u'ff61e9f5-8d44-4961-813f-e6c86fb239fe'], status=DOWN, tags=[], tenant_id=d0066f73b1db4dcabfc61318ae559960, updated_at=2018-09-29T21:41:58Z
2018-09-29 17:41:58.657 5000 DEBUG neutron.agent.dhcp.agent [req-5037b967-75f2-451b-be39-05e3e0c6fab3 6041b01c631d425d829cf96874ce50bf 671ea1986554476bbcaa620e00697d23 - - -] Calling driver for network: a3127656-d5cf-4734-b868-3ae134f10b1f action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-29 17:41:58.659 5000 DEBUG neutron.agent.linux.dhcp [req-5037b967-75f2-451b-be39-05e3e0c6fab3 6041b01c631d425d829cf96874ce50bf 671ea1986554476bbcaa620e00697d23 - - -] Building host file: /var/lib/neutron/dhcp/a3127656-d5cf-4734-b868-3ae134f10b1f/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-29 17:41:58.662 5000 DEBUG neutron.agent.linux.dhcp [req-5037b967-75f2-451b-be39-05e3e0c6fab3 6041b01c631d425d829cf96874ce50bf 671ea1986554476bbcaa620e00697d23 - - -] Done building host file /var/lib/neutron/dhcp/a3127656-d5cf-4734-b868-3ae134f10b1f/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-29 17:41:58.669 5000 DEBUG neutron.agent.linux.utils [req-5037b967-75f2-451b-be39-05e3e0c6fab3 6041b01c631d425d829cf96874ce50bf 671ea1986554476bbcaa620e00697d23 - - -] Running command (rootwrap daemon): ['kill', '-HUP', '9139'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 17:41:58.696 5000 DEBUG neutron.agent.linux.dhcp [req-5037b967-75f2-451b-be39-05e3e0c6fab3 6041b01c631d425d829cf96874ce50bf 671ea1986554476bbcaa620e00697d23 - - -] Reloading allocations for network: a3127656-d5cf-4734-b868-3ae134f10b1f reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:514
2018-09-29 17:41:58.700 5000 DEBUG neutron.agent.linux.utils [req-5037b967-75f2-451b-be39-05e3e0c6fab3 6041b01c631d425d829cf96874ce50bf 671ea1986554476bbcaa620e00697d23 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-a3127656-d5cf-4734-b868-3ae134f10b1f', 'ip', '-4', 'route', 'list', 'dev', 'tap25b2c0ed-0a'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 17:41:58.720 5000 DEBUG neutron.agent.linux.utils [req-5037b967-75f2-451b-be39-05e3e0c6fab3 6041b01c631d425d829cf96874ce50bf 671ea1986554476bbcaa620e00697d23 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-a3127656-d5cf-4734-b868-3ae134f10b1f', 'ip', '-6', 'route', 'list', 'dev', 'tap25b2c0ed-0a'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 17:41:58.741 5000 DEBUG neutron.agent.linux.utils [req-5037b967-75f2-451b-be39-05e3e0c6fab3 6041b01c631d425d829cf96874ce50bf 671ea1986554476bbcaa620e00697d23 - - -] Unable to access /var/lib/neutron/external/pids/a3127656-d5cf-4734-b868-3ae134f10b1f.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 17:41:58.743 5000 DEBUG neutron.agent.linux.utils [req-5037b967-75f2-451b-be39-05e3e0c6fab3 6041b01c631d425d829cf96874ce50bf 671ea1986554476bbcaa620e00697d23 - - -] Unable to access /var/lib/neutron/external/pids/a3127656-d5cf-4734-b868-3ae134f10b1f.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 17:41:58.744 5000 DEBUG neutron.agent.linux.external_process [req-5037b967-75f2-451b-be39-05e3e0c6fab3 6041b01c631d425d829cf96874ce50bf 671ea1986554476bbcaa620e00697d23 - - -] No process started for a3127656-d5cf-4734-b868-3ae134f10b1f disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-29 17:41:58.745 5000 DEBUG oslo_concurrency.lockutils [req-5037b967-75f2-451b-be39-05e3e0c6fab3 6041b01c631d425d829cf96874ce50bf 671ea1986554476bbcaa620e00697d23 - - -] Releasing semaphore "dhcp-agent-network-lock-a3127656-d5cf-4734-b868-3ae134f10b1f" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 17:42:35.446 5000 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-29 17:42:35.449 5000 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.003s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-29 17:42:42.700 5000 DEBUG oslo_concurrency.lockutils [req-3d36d9eb-2c61-4279-bdb6-b631b80b86c6 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - - -] Acquired semaphore "dhcp-agent-network-lock-40fff4bc-a326-419b-b1c1-a7aa26b2d41a" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 17:42:42.702 5000 DEBUG oslo_concurrency.lockutils [req-3d36d9eb-2c61-4279-bdb6-b631b80b86c6 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - - -] Releasing semaphore "dhcp-agent-network-lock-40fff4bc-a326-419b-b1c1-a7aa26b2d41a" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 17:42:44.227 5000 DEBUG oslo_concurrency.lockutils [req-3d36d9eb-2c61-4279-bdb6-b631b80b86c6 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - - -] Acquired semaphore "dhcp-agent-network-lock-40fff4bc-a326-419b-b1c1-a7aa26b2d41a" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 17:42:44.228 5000 DEBUG oslo_concurrency.lockutils [req-3d36d9eb-2c61-4279-bdb6-b631b80b86c6 c6076d0e968d46daaf0fc5019a0f8a15 bd8a53962fd64ffeab7f2932b0e688c5 - - -] Releasing semaphore "dhcp-agent-network-lock-40fff4bc-a326-419b-b1c1-a7aa26b2d41a" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
